pkgs/applications/video/backscrub/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, opencv
+, cmake
+, tensorflow-lite
+, flatbuffers
+, abseil-cpp
+}:
+stdenv.mkDerivation rec{
+  pname = "backscrub";
+  version = "0.3.0";
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "floe";
+    rev = "v${version}";
+    sha256 = "sha256-ad5arFE8JdouWvrjCwYPO34X+sDixwGfQnEUA2FUV2s=";
+    fetchSubmodules = true;
```

Comment:
Please use the version from nix.

Reply:
I can't. The version in nix does not have the CMakeFiles that the project is expecting.